cli/src/proposer/proposer.mjs

```diff
@@ -24,7 +24,7 @@ const {
   proposerPort = 8100, // Default port
   // Default Proposer URL. Client Docker needs to connect
   // to host, and this is the configured gateway IP in docker compose
-  proposerUrl = 'http://172.16.238.1',
+  proposerBaseUrl = 'http://172.16.238.1',
```

Comment:
This would be different in staging based on the staging network's IP address right?
Can this be made configureable through `config/default.mjs`?

Reply:
All the apps including proposer under CLI seems to have hardcoded variables instead of receiving them from a config or CLI variables files etc. This needs to be separate refactor on its own.